### DISCUSSION TITLE
Selecting one gpu from cli

### DISCUSSION THREAD

**Discussion by alexyalunin** [Lightning-AI/pytorch-lightning#8063] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
Hi, I have 4 gpus on my machine. I want to select the third one, that has index 2. How do I pass an argument from CLI? 
I call 
`python 4_pretrain_encoder.py --gpus 2 --max_epochs 5`
but it runs one script on two gpus instead of the third.

**Comment by tshu-w** [comment, marked as answer]
`python 4_pretrain_encoder.py --gpus 2, --max_epochs 5`

Maybe you should search first
https://github.com/PyTorchLightning/pytorch-lightning/discussions/7461
